sessionctx/variable/session.go

```diff
@@ -1339,6 +1339,11 @@ func (s *SessionVars) InTxn() bool {
 	return s.GetStatusFlag(mysql.ServerStatusInTrans)
 }
 
+// LocalTemporaryTableExists returns if there are local temporary tables in the session.
+func (s *SessionVars) LocalTemporaryTableExists() bool {
```

Comment:
I think we can remove this method because no where is using it.

Reply:
removed.